packages/cli/commands/sandbox/sync.js

```diff
@@ -0,0 +1,292 @@
+    );
+    uiLine();
+    logger.warn(i18n(`${i18nKey}.warning.standardSandbox`));
+    uiLine();
+    logger.log('');
+
+    if (!force) {
+      // Skip confirmation if force flag is present.
+      const { confirmSandboxSyncPrompt: confirmed } = await promptUser([
+        {
+          name: 'confirmSandboxSyncPrompt',
+          type: 'confirm',
+          message: i18n(`${i18nKey}.confirm.standardSandbox`, {
+            parentAccountName: getAccountName(parentAccountConfig),
+            sandboxName: getAccountName(accountConfig),
+          }),
+        },
+      ]);
+      if (!confirmed) {
+        process.exit(EXIT_CODES.SUCCESS);
+      }
+    }
+  } else {
+    logger.error('Sync must be run in a sandbox account.');
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  let initiateSyncResponse;
+
+  const baseUrl = getHubSpotWebsiteOrigin(
+    getEnv(accountId) === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD
+  );
+
+  try {
+    logger.log('');
+    spinnies.add('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.startSync`),
+    });
+
+    // Fetches sync types based on default account. Parent account required for fetch
+    const tasks = await getAvailableSyncTypes(
+      parentAccountConfig,
+      accountConfig
+    );
+
+    initiateSyncResponse = await initiateSync(
+      parentAccountId,
+      accountId,
+      tasks,
+      accountId
+    );
+
+    uiLine();
+    logger.log(
+      i18n(`${i18nKey}.info.syncStatus`, {
+        url: `${baseUrl}/sandboxes-developer/${parentAccountId}/development`,
+      })
+    );
+    uiLine();
+
+    logger.log('');
+    spinnies.succeed('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.succeed`),
+    });
+  } catch (err) {
+    trackCommandUsage('sandbox-sync', { successful: false }, accountId);
+
+    spinnies.fail('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.fail`),
+    });
+
+    logger.log('');
+    if (isMissingScopeError(err)) {
+      logger.error(
+        i18n(`${i18nKey}.failure.missingScopes`, {
+          accountName: getAccountName(parentAccountConfig),
+        })
+      );
+    } else if (err.statusCode === 429 && err.error.category === 'RATE_LIMITS') {
+      logger.error(
+        i18n(`${i18nKey}.failure.syncInProgress`, {
+          url: `${baseUrl}/sandboxes-developer/${parentAccountId}/syncactivitylog`,
+        })
+      );
+    } else if (
+      err.statusCode === 403 &&
+      err.error.category === 'BANNED' &&
+      err.error.subCategory ===
+        'sandboxes-sync-api.SYNC_NOT_ALLOWED_INVALID_USERID'
+    ) {
+      // This will only trigger if a user is not a super admin of the target account.
+      logger.error(
+        i18n(`${i18nKey}.failure.notSuperAdmin`, {
+          account: getAccountName(accountConfig),
+        })
+      );
+    } else {
+      logErrorInstance(err);
+    }
+    logger.log('');
+
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  try {
+    // Handle manual exit for return key and ctrl+c
+    process.stdin.resume();
```

Comment:
We have some utils here that may help with some of the exit handling code here: https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/lib/process.js